App/app/pubspec.yaml

```diff
@@ -31,7 +31,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
+  get_storage: ^2.1.1
```

Comment:
We are using secure storage so this is not needed, I will take care of this part.

Reply:
Won't do

Reply:
Abey do it